battDB/models.py

```diff
@@ -409,9 +414,8 @@ class Experiment(cm.BaseModel):
 
     config = models.ForeignKey(
         DeviceConfig,
-        on_delete=models.SET_NULL,
-        null=True,
-        blank=True,
+        on_delete=models.CASCADE,
+        default=1,
```

Comment:
I suspect this will require wiping the db to make sure the newly added "Single Cell" config is this default

Reply:
Ordinarily yes (or at least some manual editing), but there are no configs at all in the azure app database yet so this should work without manual interference.